src/GovUk.Education.ExploreEducationStatistics.Content.Model/GlossaryEntry.cs

```diff
@@ -8,16 +7,16 @@ public class GlossaryEntry
     {
         public Guid Id { get; set; }
 
-        public string Title { get; set; }
+        public string Title { get; set; } = string.Empty;
 
-        public string Slug { get; set; }
+        public string Slug { get; set; } = string.Empty;
 
-        public string Body { get; set; }
+        public string Body { get; set; } = string.Empty;
 
-        public DateTime Created { get; set; }
+        public DateTime Created { get; set; } = new DateTime();
 
         public Guid CreatedById { get; set; }
 
-        public User CreatedBy { get; set; }
+        public User CreatedBy { get; set; } = new User();
```

Comment:
Pretty sure you'd never want this to be assigned to a new User. If you ended up saving a new GlossaryEntry like this you could potentially also end up creating a User as part of the same save.

I'd use `public User CreatedBy { get; set; } = null!;` instead.